ec2.py

```diff
@@ -0,0 +1,119 @@
+import boto3
+import locale
+import json
+from pkg_resources import resource_filename
+
+
+# Translate between the API and what is used locally
+def translate_platform_name(operating_system, preinstalled_software):
+    os = {'Linux': 'linux', 'RHEL': 'rhel', 'SUSE': 'sles', 'Windows': 'mswin'}

```

Comment:
I prefer to have dicts split one entry per line: 
```suggestion
    os = {'Linux': 'linux',
     'RHEL': 'rhel', 
     'SUSE': 'sles', 
     'Windows': 'mswin'}
```

Reply:
Alright, I'm not really familiar with Python so I don't know the coding standard.
Fixed, thanks.